manifests/server/puppetserver.pp

```diff
@@ -76,8 +76,17 @@
   $server_max_active_instances            = $::puppet::server::max_active_instances,
   $server_max_requests_per_instance       = $::puppet::server::max_requests_per_instance,
   $server_ssl_protocols                   = $::puppet::server::ssl_protocols,
+  $server_ssl_ca_crl                      = $::puppet::server::ssl_ca_crl,
+  $server_ssl_ca_cert                     = $::puppet::server::ssl_ca_cert,
+  $server_ssl_cert                        = $::puppet::server::ssl_cert,
+  $server_ssl_cert_key                    = $::puppet::server::ssl_cert_key,
+  $server_ssl_chain                       = $::puppet::server::ssl_chain,
+  $server_crl_enable                      = $::puppet::server::crl_enable_real,
```

Comment:
Per https://docs.puppet.com/puppet/5.0/lang_reserved.html#variables using a variable with an underscore is deprecated in such a situation, so I renamed `_crl_enable` to `crl_enable_real` in `manifests/server.pp`.